catmux/window.py

```diff
@@ -61,23 +64,28 @@ def debug(self):
         for counter, split in enumerate(self.splits):
             split.debug(name=str(counter), prefix=" ")
 
-    def create(self, server_name, session_name, first=False):
+    def create(self, first=False):
         """Creates the window"""
-        tmux_wrapper = tmux.TmuxWrapper(server_name=server_name)
+        tmux_wrapper = tmux.TmuxWrapper(server_name=self.server_name)
+        target_window = ":".join([self.session_name, getattr(self, "name")])
         if not first:
-            tmux_wrapper.tmux_call(["new-window"])
-        tmux_wrapper.tmux_call(["rename-window", getattr(self, "name")])
+            tmux_wrapper.tmux_call(["new-window", "-t", self.session_name])
+        tmux_wrapper.tmux_call(
+            ["rename-window", "-t", f"{self.session_name}:$", getattr(self, "name")]
+        )
         for counter, split in enumerate(self.splits):
             if counter > 0:
                 tmux_wrapper.split()
 
             if hasattr(self, "before_commands"):
                 for cmd in getattr(self, "before_commands"):
                     tmux_wrapper.send_keys(cmd)
```

Comment:
After creating a new split, the before_commands are not targeting a specific window.
Is a new split always "in focus"?

Reply:
It is indeed not necessarily, hence [364939a](https://github.com/fmauch/catmux/pull/19/commits/364939a0c86a877f12c014577e875d53dd3ddd1d)

I've stumbled upon this today and that caused wrong behavior.